website/docs/r/container_app.html.markdown

```diff
@@ -81,9 +81,15 @@ The following arguments are supported:
 
 A `secret` block supports the following:
 
+* `identity` - (Optional) The identity to use for accessing Key Vault reference (Required if `key_vault_url` is used).
```

Comment:
Can we elaborate more here to indicate that the `identity` can be either a user assigned identity (by providing the identity id) or a system assigned identity (by providing `System`)

Reply:
updated